Assets/LeapMotionModules/Hands/Scripts/Editor/AutoRigEditor.cs

```diff
@@ -0,0 +1,23 @@
+﻿using UnityEngine;
+using System.Collections;
+using UnityEditor;
+
+namespace Leap.Unity {
+  [CustomEditor(typeof(LeapHandsAutoRig))]
+  public class ObjectBuilderEditor : Editor {
```

Comment:
The name of an editor class should match the name of the class that it edits, and should also match the name of the file it exists in.  For this case the name would be LeapHandsAutoRigEditor.